crates/data_components/src/graphql/client.rs

```diff
@@ -914,8 +952,53 @@ mod tests {
                 expected: (
                     Some(PaginationParameters {
                         resource_name: "paginatedUsers".to_owned(),
-                        args: super::PaginationArgument::First(2),
+                        pagination_argument: super::PaginationArgument::First(2),
+                        page_info_path: Some("/paginatedUsers/pageInfo".to_owned()),
+                        other_arguments: vec![],
+                    }),
+                    Some("/data/paginatedUsers/users".into()),
+                ),
+            },
+            TestPaginationParseCase {
+                name: "Pagination with other fields",
+                query: r#"
+                    query {
+                        paginatedUsers(first: 2, some_field: "value", integer_field: 10, boolean_field: true) {
+                            users {
+                                id
+                                name
+                                posts {
+                                    id
+                                    title
+                                    content
+                                }
+                            }
+                            pageInfo {
+                                hasNextPage
+                                endCursor
+                            }
+                        }
+                    }
+                "#,
+                expected: (
+                    Some(PaginationParameters {
+                        resource_name: "paginatedUsers".to_owned(),
+                        pagination_argument: super::PaginationArgument::First(2),
                         page_info_path: Some("/paginatedUsers/pageInfo".to_owned()),
+                        other_arguments: vec![
+                            super::FieldArgument {
+                                name: "some_field".to_owned(),
+                                value: "\"value\"".to_owned(),
```

Comment:
Do we want to be persisting these as `"\"value\""`, or `"value"`?

Reply:
`\"` unwraps to `"` anyway. I should probably use `r#""` to make it clearer.